api/src/main/java/ai/djl/nn/norm/BatchNorm.java

```diff
@@ -281,7 +281,7 @@ public static Builder builder() {
     }
 
     /** The Builder to construct a {@link BatchNorm}. */
-    public static final class Builder {
+    public static class Builder {
```

Comment:
What we typically do for this is create two builders: `BatchNorm.BaseBuilder` and `BatchNorm.Builder extends BatchNorm.BaseBuilder`. Then, you also have the `GhostBatchNorm.Builder extends BatchNorm.BaseBuilder`. If you do this, you won't need to override all those methods in `GhostBatchNorm.Builder` with the super.

Here are some examples. Refer to `AbstractImageFolder.ImageFolderBuilder` for an abstract or base builder. Refer to `ImageFolder.Builder` as the final builder.

There are a few changes to make. The setters in the base builders should return type `T` instead of type `Builder`. The `self` and `build` methods should go on the final builders.